packages/clay-drop-down/README.mdx

```diff
@@ -13,6 +13,7 @@ import {DropDown, DropDownWithItems} from '$packages/clay-drop-down/docs/index';
 
 -   [Composing](#composing)
     -   [Using dropdown without a trigger](#using-dropdown-without-a-trigger)
+    -   [Using dropdown in JSP files](#using-dropdown-in-jsp-files)
```

Comment:
You probably forgot to remove this one too. We have to build a plugin to generate this automatically, this is very easy to lose something.

Reply:
Yes i did forget this 🤦